cfgov/unprocessed/apps/paying-for-college/css/college-costs/college-costs.less

```diff
@@ -97,8 +97,20 @@
 			}
 
 			.financial-item + .financial-item {
-				margin-top: unit( @grid_gutter-width / 3 / @base-font-size-px, em );;
+				margin-top: unit( @grid_gutter-width / 3 / @base-font-size-px, em );
 			}
+
+			dl {
+				dt {
+			        font-size: inherit;
+			    	text-transform: none;
+			    	letter-spacing: normal;
+			    	margin-bottom: 0;
+		    	}
```

Comment:
What's going on with the closing `}` for the `dt` here? Is the indenting just off for the parent `dl`?

Reply:
I think this is just an indentation issue -- I'll push a commit that fixes the indentation in the file.